terraform/transform_import_state.go

```diff
@@ -300,13 +300,20 @@ func (n *graphNodeImportStateSub) Execute(ctx EvalContext, op walkOperation) (di
 		return diags
 	}
 
-	//EvalWriteState
```

Comment:
Import is its own, weird thing, so I felt good about reimplementing the logic behind `WriteResourceInstanceState` rather than trying to force a refactor that could handle both `graphNodeImportStateSub` and `NodeResourceAbstractInstance`.  (I also expect more import-related refactoring in the future, so I am biased towards this kind of thinking). Very open to disagreement or better suggestions!